scipy/interpolate/_interpolate.py

```diff
@@ -1333,6 +1333,49 @@ def from_spline(cls, tck, extrapolate=None):
             If bool, determines whether to extrapolate to out-of-bounds points
             based on first and last intervals, or to return NaNs.
             If 'periodic', periodic extrapolation is used. Default is True.
+
+        Examples
+        --------
+
+        This funcion only supports 1D splines out of the box.
```

Comment:
```suggestion
        This function only supports 1D splines out of the box.
```

Reply:
I'd suggest adding a very elementary example of that and emphasizing that the next example is for a more complex case. 

```
# <elementary example here>

This function only supports 1D splines out of the box, but ...
```

Reply:
Done in fa842bb